Fatal error: exception Failure("illegal argument for sin, "a=8."")
